### ISSUE TITLE
rm: cannot remove './Detection_Result/Brief_sum.txt': No such file or directory

### ISSUE THREAD

**Issue by lokdeep1993** [lxwgcool/CircMarker#3]
Hello,
I was using CircMarker for the detection of Circular RNA in my data.
Everything was fine and smooth, but then I got the error "rm: cannot remove '**./Detection_Result/Brief_sum.txt': No such file or directory**", but the file is there in the folder.
######This is the output which I was getting on the screen####
The first vChrom Size: 47
The second vChrom Size: 47
Step 3: Detect Circular RNA
rm: cannot remove './Detection_Result/*': No such file or directory
The third vChrom Size: 47
Thread Num: 47
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 
main() : creating thread 

ALL SET!!!
Combine the result together
rm: cannot remove './Detection_Result/Brief_sum.txt': No such file or directory

Any help will be appreciated.

**Comment by lxwgcool** [comment]
HI Lokdeep,

This error could be ignored. It just simply use bash to delete the old file
"Brief_sum.txt". I should add a line to check if that file existed or not.
However, it won't affect the final result.

Best
Xin

Lokdeep Teekas <notifications@github.com> 于2019年4月11日周四 下午11:17写道：

**Comment by lokdeep1993** [comment]
Hello Xin,
Thank You for your reply. I want to mention that I did not get Brief.txt and Candidate.txt as output. I got Candidate_1.txt, Candidate_2.txt and so on.
Are these my final output files?

**Comment by lxwgcool** [comment]
Hi Lokdeep,

There is no file named "brief.txt", and the final  target file is named
"brief_sum.txt".This file has already generated based on your screenshot.
I just corrected the readme file in github. Sorry to bring you this
trouble.
Actually we developed the updated version of CircMarker, which is named
CircDBG, here is the link:
https://github.com/lxwgcool/CircDBG/tree/master/CircDBG/CircDBG
The readme file has been updated for CircDBG, however i forgot to do this
same thing for CircMarker.

Have a nice working day
Best
Xin




Lokdeep Teekas <notifications@github.com> 于2019年4月12日周五 上午2:21写道：
